yarn-project/noir-contracts/src/contracts/escrow_contract/src/address_note/filter.nr

```diff
@@ -0,0 +1,22 @@
+use dep::aztec::constants_gen::MAX_READ_REQUESTS_PER_CALL;
+use dep::aztec::types::option::Option;
+
+use crate::address_note::AddressNote;
+
+fn get_owner_note(
```

Comment:
Should this be unconstrained?

Reply:
It (filter) is always executed in an unconstrained function!

Reply:
Ah, I forgot that if an unconstrained function calls a regular noir function it defaults to being an unconstrained function. Cool.